docs/userguide/promote-legacy-workload.md

```diff
@@ -0,0 +1,58 @@
+# Promote legacy workload
+
+If there is a member cluster that has deployed a workload(like Deployment) that is not managed by Karmada, we want to let Karmada to take over this workload directly and not to cause its pod to restart. To achieve this goal, We can use `karmadactl promote` command.
+
+## Example
+
+### For member cluster of `Push` mode
+There is a `nginx` deployment that belong to `default` namespace in member cluster named `cluster1`.
```

Comment:
```suggestion
There is an `nginx` Deployment that belongs to namespace `default` in member cluster `cluster1`.
```

Reply:
Thanks for your reviews!

I have a question why change `a nginx` to `an nginx`.

Reply:
Cuz it's pronounced "engine X" 😄 We use a or an based on how a word is pronounced.

Reply:
Got it, thank you!

Reply:
+1